wallet/chainntfns.go

```diff
@@ -293,6 +294,15 @@ func (w *Wallet) onBlockConnected(serializedBlockHeader []byte, transactions [][
 	w.NtfnServer.notifyMainChainTipChanged(chainTipChanges)
 	w.NtfnServer.sendAttachedBlockNotification()
 
+	// Parse vote version from extended bits.  Note that once ExtendedBits
+	// are used for other information as well this will need to be updated
+	// to accomodate.
+	voteVersion := binary.LittleEndian.Uint32(w.VoteBits().ExtendedBits)
+	if voteVersion < blockHeader.StakeVersion {
+		log.Warnf("Old vote version detected (v%v), please update your "+
+			"wallet to the latest version.", voteVersion)
```

Comment:
This message is currently misleading, it says there was an old vote version detected but prints the new version of the connected block.

We need to include both in the message too.